src/translator.js

```diff
@@ -109,7 +109,7 @@ Translator.prototype.reloadTranslations = function(type, onComplete) {
           "] (this may take a while)"
       );
       _.each(memberships, function(m) {
-        getMembershipsThrottled(m.idMember, function(err, user) {
+        getMemberships(m.idMember, function(err, user) {
```

Comment:
If I remember correctly, the throttling is required for organisations with lots of members, or we hit a rate limit. Was it causing issues?